airflow_prometheus_exporter/prometheus_exporter.py

```diff
@@ -365,6 +413,7 @@ def collect(self):
             )
         yield t_state
 
+'''
```

Comment:
This is weird, is it on purpose or just for testing?

Reply:
Commented out useless metrics to add performance, 
It is noticeable slower when it has plenty of metrics, and many of them we do not intend to use.
So i commented them.

Reply:
mmm but that is not a comment, it's a multiline string. I would actually coment it (`#`) and also leave a comment specifying why:
```python
# Removed for performance, as we ignore this metrics
```